static/js/admin.js

```diff
@@ -716,6 +728,20 @@ function _setup_page() {
                         ui.report_success(i18n.t("Any user may now invite new users!"), invite_by_admins_only_status);
                     }
                 }
+                if (response_data.inline_image_preview !== undefined) {
+                    if (response_data.inline_image_preview) {
+                        ui.report_success(i18n.t("Previews of uploaded and linked images will be shown!"), inline_image_preview_status);
+                    } else {
+                        ui.report_success(i18n.t("Previews of uploaded and linked images will not be shown!"), inline_image_preview_status);
+                    }
+                }
+                if (response_data.inline_url_embed_preview !== undefined) {
+                    if (response_data.inline_url_embed_preview) {
+                        ui.report_success(i18n.t("Previews for links will be shown!"), inline_url_embed_preview_status);
+                    } else {
+                        ui.report_success(i18n.t("Links previews will not be shown!"), inline_url_embed_preview_status);
```

Comment:
Say "Previews for linked websites will be shown!" / "Previews for linked websites will not be shown!" -- it's a lot clearer how this is different from the other setting.